drivers/clock_control/stm32f3x_ll_clock.c

```diff
@@ -46,8 +46,13 @@ void config_pll_init(LL_UTILS_PLLInitTypeDef *pllinit)
 	 * ...
 	 * 16 -> LL_RCC_PLLSOURCE_HSE_DIV_16 -> 0x0001000F
 	 */
-	pllinit->PLLDiv = (RCC_CFGR_PLLSRC_HSE_PREDIV |
-					(CONFIG_CLOCK_STM32_PLL_PREDIV1 - 1));
+	/* TODO: Enable this code and matching CONFIG symbol once one of */
```

Comment:
that comment style is weird,

use this instead...
```
/*
 *
 *
 *
 */
```

Reply:
Sure. Done.